config/cola-elmo.conf

```diff
@@ -0,0 +1,51 @@
+// Run elmo model on cola-analysis dataset 
+// 
+include "defaults.conf"
+
+
+// Do not change
+
+random_seed = 42
+word_embs_file = ""
+fastText_model_file = ""
+
+// Paths
+
+exp_name = debug  // avoid messing up other experiment records
+run_name = debug  // overwrite this when running models
+
+// Data setting
+
+reload_vocab = 1
+reload_tasks = 1
+reload_indexing = 1
+reindex_tasks = cola-analysis
+
+// Training setting
+
+load_model = 0
+
+pretrain_tasks = cola-analysis
```

Comment:
This doesn't look right. You shouldn't ever pretrain on an analysis task.

Reply:
Cola-analysis have exactly the same amount of data as cola. The only difference is it has tags. For elmo experiment, I followed the structure setting from elmo's friend, elmo + 2 layers of bi-lstm + mlp. If we turn off pretrain, the two layers bi-lstm will remain random weight.
I don't understand why we should not pretrain on cola-analysis.

Reply:
Does the cola-analysis I described fit your definition of an analysis task?

Reply:
I guess it works, it's just not the intended use. We should set up training on cola, but evaluation on cola-analysis. If that's not possible right now, though, just make it add comments here _and_ in the Python to explain that the analysis task contains the normal training set.

Reply:
Right now, jiant use different classifiers for different task, so we cannot train on cola and directly evaluate it on cola-analysis.